### PR TITLE
feat: example where closing the space always makes a new space

### PR DESCRIPTION
**Pull request by bangarang**

### CODE REVIEW

**Comment by alnoorp** [comment]
@bangarang what work needs to be done before this can be merged? Once merged we can close a support issue.

**Comment by alnoorp** [comment]
@ashleygmulligan2 is there anything holding us back from merging this PR?

**Review by ashleygmulligan2 — APPROVED** [review]

**Pull request merged**
